lbry/lbry/schema/tags.py

```diff
@@ -10,4 +10,9 @@ def normalize_tag(tag: str):
 
 
 def clean_tags(tags: List[str]):
-    return [tag for tag in set(normalize_tag(tag) for tag in tags) if tag]
+    clean = []
+    for idx, tag in enumerate(tags):
+        norm_tag = normalize_tag(tag)
+        if norm_tag and norm_tag not in clean[:idx]:
+            clean.append(norm_tag)
+    return clean
```

Comment:
What does this do differently from the original?

Reply:
Sorry, missed this. This preserves the order of the input, which is no longer necessary after changing `extend()`.

Reply:
So, it can be reverted to the original?

Reply:
Yes. To have the unit tests on the same place, how about moving the two methods from `tags.py` to `attrs.py`, and the tests from `test_tags.py` to `test_schema.py`? I don't think that the integration tests duplicate the unit tests (except in terms of coverage) - they test the same functionality of different parts. But I suppose you would prefer to remove the unit test?